src/pages/Facility/services/inventory/externalSupply/deliveryOrder/DeliveryOrderShow.tsx

```
@@ -620,6 +640,31 @@ export function DeliveryOrderShow({
+                if (value === undefined || value === null) return null;
+
+                const displayValue =
+                  field.format === "date" || field.format === "date-time"
+                    ? formatDateTime(value as string)
+                    : String(value);
+
+                return (
+                  <div key={`${field.extensionName}-${field.name}`}>
+                    <label className="text-sm font-medium text-gray-700">
+                      {field.label}
+                    </label>
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)